packages/claim-backend/docker-compose.yml

```
@@ -4,7 +4,7 @@ services:
     image: postgres
     restart: always
     ports:
-      - '${DB_PORT}:5432'
+      - '${DB_HOST:-127.0.0.1}:${DB_PORT}:5432'
```

Please assign the default port as well, else docker assigns random ports when `DB_PORT` environment variable is not specified.
```suggestion
      - '${DB_HOST:-127.0.0.1}:${DB_PORT:-5432}:5432'
```

---

Updated 👍